docs/publish/deploy-office-add-in-sso-to-azure.md

```diff
@@ -0,0 +1,253 @@
+
+1. Open your Office Add-in project in VS Code.
+1. Select the Azure logo in the [Activity Bar](https://code.visualstudio.com/docs/getstarted/userinterface). If the Activity Bar is hidden, open it by selecting **View** > **Appearance** > **Activity Bar**.
+1. In the **App Service** explorer, select **Sign in to Azure...** and follow the instructions.
+
+    :::image type="content" source="../images/azure-extension-sign-in.png" alt-text="Sign in to Azure button selected in the Azure extension.":::
+
+# [Deploy to Linux](#tab/linux)
+
+1. Right-click on App Services and select **Create new Web App**. A Linux container is used by default.
+1. Type a globally unique name for your web app and press **Enter**. The name must be unique across all of Azure and use only alphanumeric characters ('A-Z', 'a-z', and '0-9') and hyphens ('-').
+1. In Select a runtime stack, select the **Node 16 LTS** runtime stack.
+1. In Select a pricing tier, select **Free (F1)** and wait for the resources to be provisioned in Azure. When prompted to deploy, don't deploy the add-in yet. You'll do that in a later step.
+
+# [Deploy to Windows](#tab/windows)
+
+1. Right-click on App Services and select **Create new Web App... Advanced**.
+1. Type a globally unique name for your web app and press **Enter**. The name must be unique across all of Azure and use only alphanumeric characters ('A-Z', 'a-z', and '0-9') and hyphens ('-').
+1. Select the resource group you wan to use. If you don't have a resource group, select **Create a new resource group**, then enter a name for the resource group, such as *AppServiceQS-rg*.
+1. Select the **Node 16 LTS** runtime stack.
+1. Select **Windows** for the operating system.
+1. Select the location you want to serve your app from. For example, *West Europe*.
+1. Select the App Service plan you want to use. If you don't have an App Service plan, select **Create new App Service plan**, then enter a name for the plan (such as *AppServiceQS-plan*), then select **F1 Free**.
+1. For **Select an Application Insights resource for your app**, select **Skip for now** and wait the resources to be provisioned in Azure. When prompted to deploy, don't deploy the add-in yet. You'll do that in a later step.
+1. In the **App Service** explorer in Visual Studio code, expand the node for the new app, right-click **Application Settings**, and select **Add New Setting**:
+
+:::image type="content" source="../images/azure-app-service-add-setting.png" alt-text="Add app setting command.":::
+
+1. Enter `SCM_DO_BUILD_DURING_DEPLOYMENT` for the setting key.
+1. Enter `true` for the setting value.
+
+    This app setting enables build automation at deploy time, which automatically detects the start script and generates the *web.config* with it.
+
+-----
+
+1. Right-click your app service and select **Open in Portal**.
+1. When the portal opens in the browser, copy the URL of the web site from the **Overview** pane and save it. You'll need it in later steps.
```

Comment:
Consider adding this caveat, since when you copy the URL into the manifest, you're replacing spots with `https://localhost` and `api://localhost`.
```suggestion
1. When the portal opens in the browser, copy the URL of the web site from the **Overview** pane (not including the `https://` part) and save it. You'll need it in later steps.
```